diagnosis/collectlogsdvm.sh

```diff
@@ -36,7 +36,7 @@ else
 fi
 
 
-DVMCUSTOMSCRIPTLOG="/var/log/azure/acsengine-kubernetes-dvm.log"
+DVMCUSTOMSCRIPTLOG=`ls /var/log/azure/*.log | head -n 1`
```

Comment:
there might be multiple *.log file under path /var/log/azure . so the logic will fail.

Reply:
Update the code to use actual file name.